### ISSUE TITLE
Support handling of multiple fields

### ISSUE THREAD

**Issue by vpeil** [LibreCat/Catmandu-BibTeX#4]
The author/editor is handled like this:

```
author = {Einstein, Albert and Planck, Max}
```
is imported as
```
{...
    author => "Einstein, Albert and Planck, Max"
}
```
and vice versa for the export.

Wouldn't it be natural to have
```
{...
    author => ["Einstein, Albert", "Planck, Max"],
}
```
??

Backwards compatibility will be broken. Add an option for this?

**Comment by phochste** [comment]
Or support it as is. If you see an ARRAY you have multiple authors. If you have a SCALAR treat the author as is.

**Comment by vpeil** [comment]
it has been correct for the exporter (sorry for the confusion!). So this is about the importer only.

**Comment by vpeil** [comment]
solved by 437f8426620d21ae77a23d104cb1a2504b8f8401
